vigranumpy/test/test_segmentation.py

```diff
@@ -33,6 +33,49 @@ def test_labelMultiArray():
     _impl_test_labelMultiArray(numpy.uint8)
     _impl_test_labelMultiArray(numpy.uint32)
     _impl_test_labelMultiArray(numpy.float32)
+
+
+def _impl_test_applyMapping(dtype):
+    original = numpy.arange(100, dtype=dtype ).reshape(10,10)
+    mapping = { int(k) : int(k + 100) for k in original.flat[:] }
+
```

Comment:
You are right. Unfortunately, `extract` comes from boost::python, so we 
cannot easily change it.